test/models/csync_record_test.rb

```diff
@@ -0,0 +1,289 @@
+require 'test_helper'
+
+class CsyncRecordTest < ActiveSupport::TestCase
+  setup do
+    @domain = domains(:shop)
+
+    @csync_record = CsyncRecord.new
+    @csync_record.cdnskey = '257 3 13 mdsswUyr3DPW132mOi8V9xESWE8jTo0dxCjjnopKl+GqJxpVXckHAeF+KkxLbxILfDLUT0rAK9iUzy1L53eKGQ=='
+    @csync_record.action = 'initialized'
+    @csync_record.domain = domains(:shop)
+    @csync_record.last_scan = Time.now
+  end
+
+  def test_domain_must_be_present
+    @csync_record.domain = nil
+    assert_not @csync_record.valid?
+    @csync_record.domain = domains(:shop)
+  end
+
+  def test_action_must_be_present_and_valid
+    @csync_record.action = nil
+    assert_not @csync_record.valid?
+
+    @csync_record.action = 'what the fuck'
```

Comment:
Actually got the same question.
Maybe we`ll change it for some string same glibberish, but not so offensive?

Reply:
That was definitely not meant to be here, typo fixed